docs/k8s-cluster/deep-learning-examples.md

```diff
@@ -0,0 +1,170 @@
+# NVIDIA Deep Learning Examples for Tensor Cores
+
+## Introduction
+
+This repository provides State-of-the-Art Deep Learning examples that are easy to train and deploy, achieving the best reproducible accuracy and performance with NVIDIA CUDA-X software stack running on NVIDIA Volta, Turing and Ampere GPUs.
+
+## NVIDIA GPU Cloud (NGC) Container Registry
+
+These examples, along with our NVIDIA deep learning software stack, are provided in a monthly updated Docker container on the NGC container registry (https://ngc.nvidia.com). These containers include:
```

Comment:
In general, we try to keep everything in the GitHub repo in the 3rd person. Try not to use terms like "your," "you," "our,", or "we," when possible.

Reply:
Updated.